pyfixest/estimation/estimation.py

```diff
@@ -1345,6 +1409,39 @@ def _estimation_input_checks(
                 "The function argument `separation_check` must be a list of strings containing 'fe' and/or 'ir'."
             )
 
+    if vcov_kwargs is not None:
```

Comment:
I am wondering if we should keep all these checks here, or move them to the `vcov` method? Because users can call vcov post-estimation and provide new arguments, and we would like to check these as well?